capstone/src/main/webapp/script.js

```diff
@@ -170,6 +169,7 @@ function callback(results, status) {
   if (status == google.maps.places.PlacesServiceStatus.OK) {
     for (let i = 0; i < results.length; i++) {
       if (!_showSmallBusiness && !_showBlackOwnedBusiness) {
+        // put fetch here
```

Comment:
Is this a todo or left over from development?

Reply:
this code is no longer needed